phoenix-core/src/main/java/org/apache/phoenix/query/ConnectionlessQueryServicesImpl.java

```diff
@@ -109,7 +110,7 @@
  */
 public class ConnectionlessQueryServicesImpl extends DelegateQueryServices implements ConnectionQueryServices  {
     private static ServerName SERVER_NAME = ServerName.parseServerName(HConstants.LOCALHOST + Addressing.HOSTNAME_PORT_SEPARATOR + HConstants.DEFAULT_ZOOKEPER_CLIENT_PORT);
-    
+    private static final QueryServicesHelper queryServicesHelper = new QueryServicesHelper();
```

Comment:
Is it possible to push "QueryServicesHelper queryServicesHelper = new QueryServicesHelper();" to ConnectionQueryServices? Do ConnectionQueryServicesImpl and ConnectionlessQueryServicesImpl need different implementation of GuidePostsCache.